introduction.md

```diff
@@ -0,0 +1,2 @@
+Hello, I am Ariel.  Here is a bunch of information about me.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.